.github/workflows/maven-junit.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  pull_request:
+    branches: 
+      - master/**
+      - production-version/**
+      - dev-version/** 
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
are we using java 11 or java 8?

Reply:
@kachun333 which one you think is better?

Reply:
i update to use java 11 in 2cbb0d16467ad2691baade9ac7eaef0753edd010
initial i plan to run both version using [Matrix Testing](https://github.com/actions/setup-java#matrix-testing)
but due the limit of 2000 minutes/month so i think we just use version 11